requirements.txt

```diff
@@ -8,3 +8,5 @@ pyarrow==3.0.0;python_version<="3.6"
 google-cloud-bigquery
 tqdm
 deprecation
+pandas
```

Comment:
We discussed this internally, but each of these should have a guarded import in the code, where if the import fails we print something about installing the requirements.txt file.

Reply:
This is done